db/migrate/20230803133228_add_main_image_height_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddMainImageHeightToArticles < ActiveRecord::Migration[7.0]
+  def change
+    add_column :articles, :main_image_height, :integer, default: 420
```

Comment:
I got a StrongMigrations error when attempting to roll this back, asking me to ignore the column on the model and/or wrap the column add in a safety_assured. Not sure if we need to anticipate going backwards though...

Reply:
I _think_ this is a feature, not a bug of strong migrations. I.e. saving us from accidentally rolling back without explicitly marking it as safe to do so.

@maestromac care to confirm?

Reply:
Yes that's strongMigration's safety feature when removing a column.